api/src/main/java/org/openmrs/CohortMember.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import java.util.Date;
+
+public class CohortMember extends BaseOpenmrsData {
+	
+	public static final long serialVersionUID = 0L;
+	
+	protected static final Log log = LogFactory.getLog(CohortMember.class);
+
+	private Integer cohortMemberId;
+	
+	private Integer cohortId;
```

Comment:
If you change the types, you need to update the mapping file to use many-to-one for these fields, I did mention this in one of my comments in the mapping file